auth0/src/test/java/com/auth0/android/authentication/storage/CredentialsManagerTest.kt

```diff
@@ -105,7 +105,7 @@ public class CredentialsManagerTest {
     public fun shouldSaveRefreshableCredentialsUsingAccessTokenExpForCacheExpirationInStorage() {
```

Comment:
this case will never happen again, as Credentials will always have these properties passed as empty here. I'd remove this test case.